var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -31,7 +31,7 @@ class Scotch(Package):
        partitioning, graph clustering, and sparse matrix ordering."""
 
     homepage = "http://www.labri.fr/perso/pelegrin/scotch/"
-    url = "http://gforge.inria.fr/frs/download.php/latestfile/298/scotch_6.0.3.tar.gz"
+    url = "http://gforge.inria.fr/frs/download.php/latestfile/298/scotch_6.0.3.tar.gz"  # noqa: E501
```

Comment:
For the record, we ignore lines containing `https?:` when doing our flake8 line length checks.